src/components/application_manager/src/commands/command_request_impl.cc

```diff
@@ -986,6 +985,36 @@ void CommandRequestImpl::AddTimeOutComponentInfoToMessage(
   }
 }
 
+void CommandRequestImpl::AddRequestToTimeoutHandler(
+    smart_objects::SmartObject& request) const {
+  uint32_t function_id =
```

Comment:
@LitvinenkoIra what is function id is invalid_enum ( -1)

Reply:
@LuxoftAKutsan fixed 278050fc416a577487a1bd2acbf1eedfdbea3c37